caikit_nlp/modules/text_generation/peft_prompt_tuning.py

```diff
@@ -199,8 +196,8 @@ def run(
         verbalized_text = render_verbalizer(self.verbalizer, {"input": text})
         # Apply the tokenizer to the sample text & move to correct device
         tok_tensors = self.tokenizer(verbalized_text, return_tensors="pt")
-        device = PeftPromptTuning._get_device(device)
-        inputs = {k: v.to(device) for k, v in tok_tensors.items()}
+
+        inputs = {k: v.to(self.model.device) for k, v in tok_tensors.items()}
```

Comment:
FYI @rawkintrevo is actually making this change in a separate PR (it's this issue https://github.com/caikit/caikit-nlp/issues/3). Can we put it back as part of this PR and use his when it's ready instead? Since this PR is primarily targeting fine tuning anyway

Reply:
ah true. I had to make this change to make some tests pass 😄  but yes, can change it back.